README.md

```diff
@@ -7,22 +7,58 @@ Designed specifically to work with the Adafruit DHT series sensors ----> https:/
 
 Currently the library is tested with Python 2.6, 2.7, 3.3 and 3.4. It should work with Python greater than 3.4, too.
 
+Installing
+----------
+
 For all platforms (Raspberry Pi and Beaglebone Black) make sure your system is able to compile Python extensions.  On Raspbian or Beaglebone Black's Debian/Ubuntu image you can ensure your system is ready by executing:
 
 ````
 sudo apt-get update
-sudo apt-get install build-essential python-dev
+sudo apt-get install build-essential python-dev python3-dev
```

Comment:
The instructions are missing python-pip/python3-pip (at least in my Raspbian python-dev/python3-dev does not have it as dependency).

However installing python-pip or python3-pip takes care of installing build-essentials, python-dev/python3-dev and all the rest of the stuff.

As the instructions are for microcomputers, I think it's best to save as much space as possible. And in the end, the less useful software the user has installed, smaller the risk of have vulnerabilities, so it's best to separate python2 and python3.

On the other hand, I am not really sure ```sudo python -m pip install --upgrade pip setuptools wheel``` is needed for everyone. At my raspbian it's not required to install using pip (but I am not removing from the proposal anyway).

So my proposal would be:

Installing
----------

### Dependencies

For all platforms (Raspberry Pi and Beaglebone Black) make sure your system is able to compile and download Python extensions with **pip**:

On Raspbian or Beaglebone Black's Debian/Ubuntu image you can ensure your system is ready by running one or two of the following sets of commands:

Python2:
````sh
sudo apt-get update
sudo apt-get install python-pip
sudo python -m pip install --upgrade pip setuptools wheel
````

Python3:
````sh
sudo apt-get update
sudo apt-get install python3-pip
sudo python3 -m pip install --upgrade pip setuptools wheel
````

### Install with pip

Use `pip` to install from PyPI.

Python2:
```sh
sudo pip install Adafruit_DHT
```

Python3:
```sh
sudo pip3 install Adafruit_DHT
```

### Compile and install from the repository

First download the library source code from the [GitHub releases page](https://github.com/adafruit/Adafruit_Python_DHT/releases), unzipping the archive, and execute:

Python2:
```sh
cd Adafruit_Python_DHT
sudo python setup.py install
```

Python3:
```sh
cd Adafruit_Python_DHT
sudo python3 setup.py install
```

You could also git clone the repository if you want to test an unreleased version.

Reply:
I tested the instructions above, and everything seems to be working fine on Raspbian, even compiling locally from the ```remove_ez_setup``` branch.

Reply:
Thanks for the quality feedback.  I'll tweak instructions a bit and merge.